source/extensions/filters/http/jwt_authn/BUILD

```diff
@@ -9,20 +9,64 @@ load(
 envoy_package()
 
 envoy_cc_library(
-    name = "jwt_lib",
-    srcs = [
-        "extractor.cc",
-    ],
-    hdrs = [
-        "extractor.h",
+    name = "extractor_lib",
+    srcs = ["extractor.cc"],
+    hdrs = ["extractor.h"],
+    deps = [
+        "//source/common/http:utility_lib",
+        "@envoy_api//envoy/config/filter/http/jwt_authn/v2alpha:jwt_authn_cc",
     ],
+)
+
+envoy_cc_library(
+    name = "jwks_cache_lib",
+    srcs = ["jwks_cache.cc"],
```

Comment:
I can't spot this on the CI coverage report (https://53860-65214191-gh.circle-artifacts.com/0/build/envoy/generated/coverage/coverage.html), do you know why?

Reply:
Your coverage maybe from other PR coverage tests.   I don't know how to check show coverage from PR coverage test.  But from the test log it has  jwks_cache.html which should reflect jwks_cache code coverage.